Firestore/Example/Tests/Local/FSTMutationQueueTests.mm

```diff
@@ -243,8 +243,7 @@ - (void)testAllMutationBatchesAffectingDocumentKey {
                        @{ @"a" : @1 }),
     FSTTestSetMutation(@"foo/bar",
                        @{ @"a" : @1 }),
-    FSTTestPatchMutation(@"foo/bar",
-                         @{ @"b" : @1 }, nil),
+    FSTTestPatchMutation("foo/bar", {{"b", "1"}}, {}),
```

Comment:
This change isn't okay. We need to be able to continue to construct internal field values with non-string types.

This may mean initializing this map with pairs of string to FieldValue in the short term.

Alternatively leave FSTTestPatchMutation's interface alone (leaving these parts as Objective-C) and use objective-C instanceof checks to select the right FieldValue under the covers.